features/ui/footer/footer.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import styled from "styled-components";
+import { color, space, breakpoint, textFont } from "@styles/theme";
+import Link from "next/link";
+
+const Container = styled.footer`
+  background-color: ${color("gray", 50)};
+  height: 11rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  padding-inline: 0;
+
+  @media (min-width: ${breakpoint("desktop")}) {
+    flex-direction: row;
+    justify-content: space-between;
+    height: 3.7rem;
+    padding-inline: ${space(8)};
+  }
+`;
+
+const Version = styled.p`
+  color: ${color("gray", 400)};
+  order: 3;
+  padding: 0;
+  margin: 0;
+
+  @media (min-width: ${breakpoint("desktop")}) {
+    order: 0;
+    padding: initial;
+    margin: initial;
+  }
+`;
+
+const Nav = styled.div`
+  margin-right: ${space(16)};
+  order: 1;
+  margin: 0 auto;
+
+  @media (min-width: ${breakpoint("desktop")}) {
+    order: 0;
+    margin-right: 0;
+  }
+`;
+
+const List = styled.ul`
+  margin: 0;
+  padding: 0;
+  display: flex;
+  gap: ${space(6)};
+  list-style-type: none;
+  width: 100%;
+
+  @media (min-width: ${breakpoint("desktop")}) {
+    width: 18rem;
```

Comment:
You use a fixed width a couple of times here. I'm wondering if that's necessary. Usually, I'd try to use fixed dimensions as seldomly as possible. I'll try to have a closer look during the week